services/src/main/scala/cromwell/services/metadata/impl/MetadataSummaryRefreshActor.scala

```diff
@@ -21,7 +22,7 @@ object MetadataSummaryRefreshActor {
   case object MetadataSummarySuccess extends MetadataSummaryActorMessage
   final case class MetadataSummaryFailure(t: Throwable) extends MetadataSummaryActorMessage
 
-  def props() = Props(new MetadataSummaryRefreshActor())
+  def props() = Props(new MetadataSummaryRefreshActor()).withDispatcher(ServiceDispatcher)
```

Comment:
I think this should be something else. We don't want the summarizer gumming up the rapid API responses